src/utils.ts

```diff
@@ -2,6 +2,8 @@ import type { Material, MaterialStyleFlags, MaterialStyleValues } from './types/
 import type { AutoStyleVariant, NumericStyleVariant, StyleVariant } from './types/styles';
 import type { DriveColorStrength, VariableTransitions } from './types/transitions';
 
+import type { ProjectColor } from '.';
```

Comment:
Should the `AnyColor` constant and type be defined here to avoid a circular import? Similar to how we define RGB and RGBA in this file